test/AbstractMethodError.test.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
```

Comment:
This is a unit test and file should be named `.unittest.js`

Reply:
Done. Didn't find info about it in `~/test/README.md`